cuda/cuda-test/inc/cuda_properties_iterator.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "cuda_runtime.h"
+#include <algorithm>
+
+class m_CudaProperties {
```

Comment:
why m_ prefix for class name. keep naming conventions consistent. if you use camelcase, use it everywhere